packages/manager/src/features/CloudPulse/Alerts/CreateAlert/GeneralInformation/AlertSeveritySelect.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import { Controller, useFormContext } from 'react-hook-form';
+
+import { Autocomplete } from 'src/components/Autocomplete/Autocomplete';
+
+import { alertSeverityOptions } from '../../constants';
+
+import type { AlertSeverityType } from '@linode/api-v4';
+export interface CloudViewRegionSelectProps {
+  /**
+   * name used for the component in the form
+   */
+  name: string;
+}
+
+export const CloudPulseAlertSeveritySelect = (
+  props: CloudViewRegionSelectProps
+) => {
+  const { name } = props;
+
+  const { control } = useFormContext();
```

Comment:
```suggestion
  const { control } = useFormContext<CreateAlertDefinitionPayload>();
```

We must provide a generic type so that `react-hook-form` knows the types of the form state. Right now, if you look at `field.value` on line 44, it is type `any`. We need to improve that. 

Once this change is made, you'll realize that `name` isn't type-safe enough.